cmd/kubelet/app/server.go

```diff
@@ -95,6 +96,7 @@ type KubeletServer struct {
 	TLSPrivateKeyFile              string
 	CertDirectory                  string
 	NodeStatusUpdateFrequency      time.Duration
+	ChaosChance                    float64
```

Comment:
My concern is with "spinkling of test code" throughout the codebase. I'd prefer to have them separated out in an easily discernible way, in case we wish to rip them out or refactor them in the future. At the least, please comment.